docs/src/main/asciidoc/smallrye-kafka-incoming.adoc

```diff
@@ -0,0 +1,148 @@
+
+Type: _boolean_ | false | `true`
+
+| *health-readiness-enabled* | Whether readiness health reporting is enabled (default) or disabled
+
+Type: _boolean_ | false | `true`
+
+| *health-readiness-topic-verification* | _deprecated_ - Whether the readiness check should verify that topics exist on the broker. Default to false. Enabling it requires an admin connection. Deprecated: Use 'health-topic-verification-enabled' instead.
+
+Type: _boolean_ | false | 
+
+| *health-readiness-timeout* | _deprecated_ - During the readiness health check, the connector connects to the broker and retrieves the list of topics. This attribute specifies the maximum duration (in ms) for the retrieval. If exceeded, the channel is considered not-ready. Deprecated: Use 'health-topic-verification-timeout' instead.
+
+Type: _long_ | false | 
+
+| *health-topic-verification-enabled* | Whether the startup and readiness check should verify that topics exist on the broker. Default to false. Enabling it requires an admin client connection.
+
+Type: _boolean_ | false | `false`
+
+| *health-topic-verification-timeout* | During the startup and readiness health check, the connector connects to the broker and retrieves the list of topics. This attribute specifies the maximum duration (in ms) for the retrieval. If exceeded, the channel is considered not-ready.
+
+Type: _long_ | false | `2000`
+
+| *tracing-enabled* | Whether tracing is enabled (default) or disabled
+
+Type: _boolean_ | false | `true`
+
+| *cloud-events* | Enables (default) or disables the Cloud Event support. If enabled on an _incoming_ channel, the connector analyzes the incoming records and try to create Cloud Event metadata. If enabled on an _outgoing_, the connector sends the outgoing messages as Cloud Event if the message includes Cloud Event Metadata.
+
+Type: _boolean_ | false | `true`
+
+| *topics* | A comma-separating list of topics to be consumed. Cannot be used with the `topic` or `pattern` properties
+
+Type: _string_ | false | 
+
+| *pattern* | Indicate that the `topic` property is a regular expression. Must be used with the `topic` property. Cannot be used with the `topics` property
+
+Type: _boolean_ | false | `false`
+
+| *key.deserializer* | The deserializer classname used to deserialize the record's key
+
+Type: _string_ | false | `org.apache.kafka.common.serialization.StringDeserializer`
+
+| *value.deserializer* | The deserializer classname used to deserialize the record's value
+
+Type: _string_ | true | 
+
+| *fetch.min.bytes* | The minimum amount of data the server should return for a fetch request. The default setting of 1 byte means that fetch requests are answered as soon as a single byte of data is available or the fetch request times out waiting for data to arrive.
+
+Type: _int_ | false | `1`
+
+| *group.id* | A unique string that identifies the consumer group the application belongs to. If not set, a unique, generated id is used
+
+Type: _string_ | false | 
```

Comment:
We need to figure out this sync issue, because everytime we sync from SmallRye, we lose this important piece of content:

```
| *group.id* | A unique string that identifies the consumer group the application belongs to.

If not set, defaults to the application name as set by the `quarkus.application.name` configuration property.

If that is not set either, a unique, generated id is used.
It is recommended to always define a `group.id`, the automatic generation is only a convenient feature for development.

Type: _string_ | false |
```

(We should actually update this to mention `${quarkus.uuid}`, but first, we need to make sure we don't lose the text all the time.)

Reply:
I tested a "not-so-ugly" way of doing this. But it involves generating optional includes in the original SmallRye RM documentation. @Ladicek what do you think?

Reply:
I'm not sure if it needs to be includes, maybe just AsciiDoc conditionals (`ifdef`, `ifndef`) would be enough?

Reply:
I don't think that neither `include` nor `ifdef` solves this issue. I don't think that we should add quarkus related content to the SmallRye RM repo.

Reply:
I included the `${quarkus.uuid}` to `group.id` property documentation.

Reply:
Well we already have Quarkus-specific stuff in RM documentation, so I would be fine with including more, but your call :-)

Reply:
I don't know I can hardly find any reference to Quarkus in RM repo. Maybe we can decide on this later?

Reply:
Sure thing!